release.py

```diff
@@ -129,21 +127,24 @@ def parse_args(argv):
                         help='no-sanity-checks; i.e. skip sanity checks commands')
     parser.add_argument('--no-generate-source-file', dest='no_source_file', action='store_true', default=False,
                         help='Do not generate source file when building')
-    parser.add_argument('--ignition-repo', dest='ignition_repo', action='store_true', default=False,
-                        help='use ignition robotics URL repositories instead of OSRF')
+    parser.add_argument('--no-ignition-auto', dest='no_ignition_auto', action='store_true', default=False,
+                        help='Use package name to create --package-alias if package name starts with ign-')
     parser.add_argument('--upload-to-repo', dest='upload_to_repository', default="stable",
                         help='OSRF repo to upload: stable | prerelease | nightly')
     parser.add_argument('--extra-osrf-repo', dest='extra_repo', default="",
                         help='extra OSRF repository to use in the build')
     parser.add_argument('--nightly-src-branch', dest='nightly_branch', default="main",
                         help='branch in the source code repository to build the nightly from')
     args = parser.parse_args()
-    if not args.package_alias:
-        args.package_alias = args.package
+
+    args.package_alias = args.package
+    # If ignition auto is enabled, replace ign- at the begging of the string
```

Comment:
nit: "replace ign- at the begging" --> "replace ign- with ignition- at the beginning"

Reply:
38ac800